kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.cdc.core.importer;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.apache.shardingsphere.data.pipeline.cdc.core.importer.CDCImporterManager.getImporter;
```

Comment:
<img width="1073" alt="image" src="https://github.com/apache/shardingsphere/assets/101622833/19e541e7-63c0-4ee7-8eaa-3034fa1385b6">

It also looks like it violates the checkstyle, 
> Only those that relate to Mockito, junit Assertions, hamcrest CoreMatchers and MatcherAssert can use static import.

Reply:
my mistake, sorry for not adhering to some of the checkstyle rules in my recents code submissions! I've now corrected it here https://github.com/apache/shardingsphere/pull/28666/commits/a8c659f7ff1112b0555506475d3fef4db36440c0

if there's something else, let me know :)

Reply:
Ok, there is one more, I think this should be the last one :)
> https://github.com/apache/shardingsphere/actions/runs/6456714607/job/17526789037?pr=28666
Run 'mvn spotless:apply' to fix these violations.

Reply:
for some reason i'm not able to run the command `mvn spotless:apply`
![image](https://github.com/apache/shardingsphere/assets/85903568/85fea802-b883-4ebc-9c1e-c78f5bd41a4e)

is there something I'm missing here?

Reply:
<img width="799" alt="image" src="https://github.com/apache/shardingsphere/assets/101622833/c1497418-db6d-4455-9b90-e45a01575648">

`./mvnw spotless:apply -Pcheck` It needs to be executed in the root directory of the project

Reply:
It worked, thank you very much for the help!

Reply:
you're welcome, thank you for your contribution.